server/graphql/v2/object/Collective.js

```diff
@@ -10,6 +10,53 @@ export const Collective = new GraphQLObjectType({
   fields: () => {
     return {
       ...AccountFields,
+      stats: {
+        type: Stats,
+        resolve(collective) {
+          return collective;
+        },
+      },
+      host: {
+        description: 'Get the host collective that is receiving the money on behalf of this collective',
+        type: Account,
+        resolve: resolveHost,
+      },
+    };
+  },
+});
+
+function resolveHost(collective, _, req) {
+  if (collective.HostCollectiveId) {
+    return req.loaders.collective.findById.load(collective.HostCollectiveId);
+  }
+  if (collective.ParentCollectiveId) {
+    return req.loaders.collective.findById.load(collective.ParentCollectiveId).then(parentCollective => {
+      if (parentCollective && parentCollective.HostCollectiveId) {
+        return req.loaders.collective.findById.load(parentCollective.HostCollectiveId);
+      } else {
+        return null;
+      }
+    });
+  }
+  return null;
+}
+
+const Stats = new GraphQLObjectType({
```

Comment:
I don't like this object. I guess it is to mock the API v1? Why not having balance as property of the Collective itself?

Reply:
Yeah, I just mocked API V1 there. Let me check how that `stats` is used in other places on V1. If I don't find any problems I'll add the balance to the Collective itself.

Reply:
I checked and the stats type from api v1 is a little bit bigger that the one I create here (I only copied what I needed) but yeah, it makes sense for a collective to have its balance a part of its fields. I'll move it and delete the type. As the migration progress we can decide where to put the rest of the stats fields.

Reply:
Done!